ci/Dockerfile.ci-eoan

```diff
@@ -1,13 +1,13 @@
-FROM ubuntu:disco-20190204
+FROM ubuntu:eoan-20200313
 
 # Build deps
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install \
     ccache \
-    clang-7 \
+    clang-9 \
     clang-tidy-8 \
```

Comment:
Should we update clang-tidy here, too?

Reply:
I actually wanted to do the tool upgrades later. Switching to the new compilers was not avoidable, though. In the case of clang-tidy, there seem to be some new issues with version 9, so I'd prefer doing the upgrade separately.